### ISSUE TITLE
A11y_SQL database projects extension_View And More Action_Update Project From Database_Name Role Value: Name is not descriptive for the radio buttons which are present in the "Action" section.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/azuredatastudio#22098]
### GitHub Tags
#A11yTCS; #A11ySev3; #DesktopApp; #Name Role Value; #WCAG4.1.2; #Win11; #SH-SQL Azure Data Studio-Win32-March23; #Win32; #SQL Azure Data Studio; #NVDA; #Screenreader; #A11yWCAG2.1; #Regressed:04-03-23; #Closed;

### Environment Details
Application Name: SQL Azure Data Studio
Version: 1.41.2 (user setup)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Screen reader: NVDA

### Repro Steps

1. Launch Azure Data Studio
2. Tab till extension and search for “SQL Database Projects” and install it.
3. Tab till database project and press enter. “Database Project” panel will open.
4. Tab till “View and More Action” button and press enter.
5. Tab till “Update Project from Database” and press enter.
6. Tab till radio buttons which are present in the "Action" section.
7. Verify whether the name is descriptive for the radio buttons which are present in the "Action" section or not.

### Actual Result:​
Name is not descriptive for the radio buttons which are present in the "Action" section.

### Note issue
Similar issue observed in both update and create project from database popups.
Field names are not descriptive. (Screen reader is not reading the section/grouping information)

### Expected Result:​
Name should be more descriptive for the radio buttons which are present in the "Action" section.
Screen reader must read the name of the radio button along with the section name as action.

### User Impact:
Screen reader user will not be able to get the proper information about the radio buttons.

### Attachment:
![Name Role Value](https://user-images.githubusercontent.com/99243079/222406529-2ce30031-960e-42ab-84b0-9094474fdcbe.gif)
![Des](https://user-images.githubusercontent.com/99243079/223638085-bc62da15-fcc0-499c-adc3-1526f64bb553.gif)

**Comment by kisantia** [comment]
@Samhitha66 this has been fixed in the latest version of the sql database projects extension (0.23.0). Please download the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch) and verify this is fixed. Thanks!

**Comment by Samhitha66** [comment]
Regressed the issue in the below environment
Version: 1.43.0-insider (user setup)
Commit: https://github.com/microsoft/azuredatastudio/commit/afafee844ce2b07eb55bdb6f40ee1ce88c0bbf4e
Date: 2023-03-29T05:28:54.263Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
![image](https://user-images.githubusercontent.com/99243079/228760186-bdd13f7c-24ae-4910-8d56-953222bef05c.png)
The screen reader is not reading the section grouping names ex: Source database, Target project , settings etc
![Bug_22098_Not fixed](https://user-images.githubusercontent.com/99243079/228760474-569bb306-b0d5-4ff9-9b94-dae873fdd29c.gif)

The issue is not fixed. Please find the attachment for the reference.

**Comment by kisantia** [comment]
@Samhitha66 the original issue of "Name is not descriptive for the radio buttons which are present in the "Action" section" is now fixed. Please create a separate issue for section groupings not being announced.

![radioGroupAriaLabel](https://user-images.githubusercontent.com/31145923/227607344-84ba249d-cd54-4c27-93f9-09430fea9d5f.gif)

**Comment by Samhitha66** [comment]
@kisantia 
As pert the above comment logged a new bug for the section grouping issue.
New bug : https://github.com/microsoft/azuredatastudio/issues/22583
Hence closing this bug.

**Comment by kisantia** [comment]
Great, thank you!
